.github/workflows/e2e.generic.schedule.main.default.slsa2.yml

```diff
@@ -4,6 +4,10 @@ on:
     - cron: "0 3 * * *"
   workflow_dispatch:
 
+env:
+  GH_TOKEN: ${{ secrets.E2E_GO_TOKEN }}
```

Comment:
feel free to create another token and rename it accordingly. We need to create an account specifically for these. The existing token is mine.

Reply:
Yeah. I wasn't sure what permissions to attach to this or why it's necessary over just using the built-in `GITHUB_TOKEN`. Does it need permissions not available on `GITHUB_TOKEN`?
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
I added a comment to #25 to fix this there.

Reply:
it's needed for the `gh` CLI utility we use, e.g. to create a release, etc

Reply:
Would the workflow's normal `GITHUB_TOKEN` not work?

Reply:
in some cases it failed, but I don't remember for which :/ 
I used it at some point and reverted to the PAT...

Reply:
OK, I'll play around with it and see if I can't document what's needed.